Assets/Game/Scripts/Goal.cs

```diff
@@ -0,0 +1,39 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.SceneManagement;
+
+public class Goal : MonoBehaviour
```

Comment:
please wire this class with the other things in the game. For example, make sure it triggered when the player get the ring and publish an event that the GameFlowManager listen (the scene will be loaded by the GameFlowManager)